pkg/messages/leaveFoodMessage.go

```diff
@@ -0,0 +1,19 @@
+package messages
+
```

Comment:
Change 'leaveFoodMessage' to 'leaveFoodAmtMessage'. (I think its more descriptive)